doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -135,13 +133,7 @@ The procedure to install CORTX on OVA is mentioned below.
 
    ::
 
-    systemctl status rabbitmq-server 
-    systemctl status elasticsearch   
-    systemctl status haproxy
-    systemctl status s3authserver 
-    systemctl status sspl-ll      
-    systemctl status csm_agent    
-    systemctl status csm_web
+    systemctl status rabbitmq-server elasticsearch haproxy s3authserver sspl-ll csm_agent csm_web hare-consul-agent.service
```

Comment:
Could you please remove `.service` from `hare-consul-agent` service?

Reply:
done